### DISCUSSION TITLE
equalise note size for 5 key mania with argon.

### DISCUSSION THREAD

**Discussion by sourgummyes** [ppy/osu#20777] [Ideas] [answered]
With the new osu!lazer argon skin, mania finally got it's reskin. 
Though 5 key has been somewhat butchered with these new version, in Triangles and Classic, all the notes are equal (or near equal) in size, but with Argon the center key is now this gigantic key.

**Comment by mk56-spn** [comment, marked as answer]
https://github.com/ppy/osu/discussions/20624 duplicate
